src/example/app.js

```diff
@@ -67,6 +91,21 @@ function App() {
          <ProductContainer>
             <NotifyProduct sku={1453074} salesChannelID={1} />
          </ProductContainer>
+         <hr />
+         <StyledStoreDropdown
+            stores={stores}
+            onClickForStore={store => () => {
+               console.log(store);
+            }}
+         />
+         <hr />
+         <StyledLanguageDropdown
+            languages={languages}
+            translationPreferencesUrl="https://zmitchell.cominor.com/Login"
```

Comment:
I don't think you should include your cominor url here. Either www.cominor.com or www.ifixit.com. If someone's testing it, they can edit it themselves.

Reply:
Didn't mean to leave that there. I edited it in my rebase.